dockerfiles/galaxy-kickstart-metavisitor/Dockerfile-metavisitor

```diff
@@ -2,15 +2,5 @@ FROM artbio/galaxy-kickstart-metavisitor-base
 
 MAINTAINER Marius van den Beek <m.vandenbeek@gmail.com>
 
-# Expose port 80 (webserver), 21 (FTP server), 8800 (Proxy), 9002 (supvisord web app)
-EXPOSE :80
-EXPOSE :21
-EXPOSE :8800
-EXPOSE :9002
-
 # Mark folders as imported from the host.
 VOLUME ["/export", "/var/lib/docker"]
```

Comment:
OK, that will be the next PR addressing https://github.com/ARTbio/ansible-artimed/issues/140